vms/platformvm/txs/builder/builder.go

```diff
@@ -661,3 +670,28 @@ func (b *builder) NewTransferSubnetOwnershipTx(
 	}
 	return tx, tx.SyntacticVerify(b.ctx)
 }
+
+func (b *builder) NewBaseTx(
+	amount uint64,
+	keys []*secp256k1.PrivateKey,
+	changeAddr ids.ShortID,
+) (*txs.Tx, error) {
+	ins, outs, _, signers, err := b.Spend(b.state, keys, amount, b.cfg.TxFee, changeAddr)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't generate tx inputs/outputs: %w", err)
+	}
+
+	utx := &txs.BaseTx{
+		BaseTx: avax.BaseTx{
+			NetworkID:    b.ctx.NetworkID,
+			BlockchainID: b.ctx.ChainID,
+			Ins:          ins,
+			Outs:         outs,
+		},
+	}
+	tx, err := txs.NewSigned(utx, txs.Codec, signers)
+	if err != nil {
+		return nil, err
+	}
+	return tx, tx.SyntacticVerify(b.ctx)
+}
```

Comment:
This function seems to just burn `amount`.... That seems unexpected... Can we transfer `amount` to some `owner`?